examples/timeline-gantt.py

```diff
@@ -0,0 +1,149 @@
+import dash_bootstrap_components as dbc
+import pandas as pd
+import plotly.figure_factory
+from dash import Dash, html, dcc, Input, Output, dash_table, ctx
+import plotly.express as px
+
+DATA_TABLE_SCHEMA = [
+    {
+        "id": "Task",
+        "name": "Task",
+    },
+    {
+        "id": "Duration",
+        "name": "Days",
+        "type": "numeric",
+    },
+    {"id": "Resource", "name": "Resource", "presentation": "dropdown"},
+    {"id": "Start", "name": "Start", "type": "datetime"},
+    {"id": "Finish", "name": "End", "type": "datetime", "editable": False},
+]
+
+DATA_TABLE_STYLE = {
+    "style_data_conditional": [
+        {"if": {"column_id": "Finish"}, "backgroundColor": "#eee"}
+    ],
+    "style_header": {
+        "color": "white",
+        "backgroundColor": "#799DBF",
+        "fontWeight": "bold",
+    },
+}
+
+# Default new row for datatable
+new_task_line = {
+    "Task": "",
+    "Start": "2016-01-01",
+    "Duration": 0,
+    "Resource": "A",
+    "Finish": "2016-01-01",
+}
+df_new_task_line = pd.DataFrame(new_task_line, index=[0])
+
+
+def get_default_table() -> pd.DataFrame:
+    """
+    Pull a csv file from Plotly datasets directory and return it as a default DataFrame for the app.
+    """
```

Comment:
This comment is unnecessary because the code on the next line very readable `return pd.read_csv...`